modules/tester/tester.go

```diff
@@ -0,0 +1,18 @@
+package tester
+
+import (
+	"os"
+	"path/filepath"
+)
+
+const (
+	SiaTestingDir = "SiaTesting"
+)
+
+// TempDir returns the tempified version of the input directory, after creating
+// the tempified directory.
+func TempDir(dir string) string {
+	temp := os.TempDir()
+	tempified := filepath.Join(temp, SiaTestingDir, dir)
+	return tempified
```

Comment:
yeah, just define `var siaTempDir = filepath.Join(os.TempDir(), SiaTestingDir)` and then `TempDir(dir)` just becomes `return filepath.Join(siaTempDir, dir)`